### ISSUE TITLE
Class of button "Send"

### ISSUE THREAD

**Issue by carlosanpardo** [MuhammadHassanRazaa/WhatsAppSender#6]
Hello. I reviewed your code and the class of the button "send" is outdated. 

`wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementToBeClickable(By.ClassName("_1U1xa"))).Click();`

I changed the classname "_1U1xa" to "_4sWnG" and works...

How We can get the current button class to send the message in the futures WP updates?

Thank yoy for your help.

**Comment by MuhammadHassanRazaa** [comment]
Thank you!

Could you please create a pull request and change the class?

I don’t know how can we get the updated class, maybe we can check it with some sorta string matching or icon etc, am really not sure, will look into it.

**Comment by carlosanpardo** [comment]
Hello!. Maybe we can send the message if search by the below html data elements:

`<span data-testid="send" data-icon="send"......`

What do you think @MuhammadHassanRazaa ?

Cheers.

**Comment by MuhammadHassanRazaa** [comment]
yeah, you're right, but it can also be changed in future. Could you please update it by creating pull request?
